app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">More about teachers' salaries and benefits</h2>
+
+    <div class="inset">
+      <p>
+        Your teaching salary will depend on your role in the school and qualifications.
+        There are lots of opportunities to increase your salary during your career:
+      </p>
+
+      <ul>
+        <li>teachers who make a sustained contribution to their school could expect to earn £40, 625 to £43,685 or more in London</li>
```

Comment:
@philspray just a small point - might it be a bit clearer/impactful if we rounded these numbers to make them easier to read? e.g. teachers who make a sustained contribution to their school could expect to earn around £40k to £43k or more in London (but feel free to ignore!)